shippable.yml

```diff
@@ -0,0 +1,34 @@
+language: go
+
+go:
```

Comment:
Said PR merged, so it might be worth copying the changes here. Sorry for the PR race.